include/exampleConsumeCb.hpp

```diff
@@ -0,0 +1,14 @@
+#ifndef CVDP_EXAMPLE_CONSUME_CB_H
+#define CVDP_EXAMPLE_CONSUME_CB_H
+
+#include "librdkafka/rdkafkacpp.h"
+
+/**
+ * NOTE: This is supposed to be a little more efficient.
```

Comment:
More efficient than what?

Reply:
Also this same ExampleConsumeCb class seems to live in rdkafka_example.cpp - do we need it in both places?

Reply:
This is an old comment, I'm not sure what it is referencing. The class that lives in rdkafka_example.cpp is separate from the project and that file is marked as deprecated in #29.